CONTRIBUTING.md

```diff
@@ -15,6 +15,10 @@ Look through currently [open pull requests](https://github.com/ansible-collectio
 You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. Also, consider taking up a valuable, reviewed, but abandoned pull request which you could politely ask the original authors to complete yourself.
 And it is always worth saying that good reviews are often more valuable than pull requests themselves.
 
+## Discussions
+
+For open questions, broad suggestions and other comments that won't typically fit in the scope of an issue or pull request, [discussions](https://github.com/ansible-collections/community.mysql/discussions) are available. That section provides a place to have a more open and informal conversation about any and all things related to this collection, included but not limited to future development plans, functionality explanations and feature proposals that are not yet fleshed out enough for an issue of their own.
```

Comment:
```suggestion
For open questions, broad suggestions, and other comments that will not typically fit in the scope of an issue or pull request, [discussions](https://github.com/ansible-collections/community.mysql/discussions) are available. That section provides a place to have a more open and informal conversation about any and all things related to this collection, included but not limited to future development plans, functionality explanations and feature proposals that are not yet fleshed out enough for an issue of their own.
```
1) I'm not an expert but I think the comma should be there
2) In such documents, better to avoid abbreviations. It's described in Ansible style guide. If interesting, here it is (and there are a few references at the bottom of the page) https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#style-guide

Reply:
Oxford comma, you're right.